sale_invoice_date_from_picking_actual_date/models/res_company.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    use_actual_date_for_invoice = fields.Boolean()
```

Comment:
To make the usage of the field clearer.
```suggestion
    use_picking_actual_date_for_invoice = fields.Boolean()
```